package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@chargebee/chargebee-js-types",
+  "version": "1.0.0",
+  "description": "TypeScript Definitions for Chargebee JS SDK",
+  "author": "",
+  "license": "MIT",
+  "scripts": {},
+  "typeScriptVersion": "4.8.4",
```

Comment:
Can we include all compatible ts versions? Not all customers will be using the latest ts version.

Reply:
As discussed, [official](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions) param name if `typesVersions` and not `typeScriptVersion`.
Removed incorrect one.